CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## dev
 
+- Update link to troubleshooting page in bug report template
```

Comment:
We don't need a changelog entry here since this change has no effect on end users.

Reply:
I have removed it.

It has an effect on all end users that report a bug and try to follow the link.

Something in the template to suggest that the check box is optional would avoid this sort of back and forward in future.